WordPress/Tests/WhiteSpace/ControlStructureSpacingUnitTest.inc

```diff
@@ -45,3 +45,31 @@ endif;
 if ( false ) :
 else :
 endif;
+
+function($arg){} // Bad
+function ( $arg ) {
```

Comment:
WPCS for PHP doesn't actually specify a space before `(` or not for closures simply because they aren't being used in core. For WPCS for JS, a space should not be added, but for PSR-2, a space is needed. The powers that be may go either way - stay inline with the JS, or follow PSR-2.

----

Can checks for `use ()` be added too? e.g.

~~~php
$closureWithArgsAndVars = function ( $arg1, $arg2 ) use ( $var1, $var2 ) {}; // OK

$closureWithArgsAndVars = function ( $arg1, $arg2 ) use( $var1, $var2 ) {}; // Bad, no space before open parenthesis prohibited.
$closureWithArgsAndVars = function ( $arg1, $arg2 ) use  ( $var1, $var2 ) {}; // Bad, expected exactly one space before opening parenthesis.

$closureWithArgsAndVars = function ( $arg1, $arg2 ) use ( $var1, $var2) {}; // Bad, no space before closing parenthesis prohibited
$closureWithArgsAndVars = function ( $arg1, $arg2 ) use ( $var1, $var2  ) {}; // Bad, expected exactly one space before closing parenthesis

$closureWithArgsAndVars = function ( $arg1, $arg2 ) use ($var1, $var2 ) {}; // Bad, no space after open parenthesis prohibited
$closureWithArgsAndVars = function ( $arg1, $arg2 ) use (  $var1, $var2 ) {}; // Bad, expected exactly one space after opening parenthesis

$closureWithArgsAndVars = function ( $arg1, $arg2 ) use ( $var1, $var2 ){}; // Bad, space between closing parenthesis and control structure required.
$closureWithArgsAndVars = function ( $arg1, $arg2 ) use ( $var1, $var2 )  {}; // Bad, expected exactly one space between closing parenthesis and control structure.
~~~